api/breaker/breaker.go

```diff
@@ -292,7 +292,7 @@ func New(cfg Config) (*CircuitBreaker, error) {
 	}
 
 	cb := CircuitBreaker{cfg: cfg}
-	cb.nextGeneration(cfg.Clock.Now())
+	cb.nextGeneration(cfg.Clock.Now().UTC())
```

Comment:
Was this supposed to be UTC all along and we missed it? Could this change cause problems?

Reply:
I don't think it matters except for the equality checks in the test. Reverted this and fixed the test instead